components/raftstore/src/store/msg.rs

```diff
@@ -377,6 +377,10 @@ pub enum CasualMessage<EK: KvEngine> {
     RejectRaftAppend {
         peer_id: u64,
     },
+    RefreshRegionBuckets {
+        region_epoch: RegionEpoch,
+        region_buckets: metapb::Buckets,
```

Comment:
How about using `regions_buckets: Vec<Vec<u8>>,` instead and constructing the `metapb::Buckets` in `on_refresh_region_buckets()`? You can add the start/end key(now there are two places doing it) and set the bucket version here.

Reply:
agree.